californium-core/src/main/java/org/eclipse/californium/core/network/BaseMatcher.java

```diff
@@ -83,9 +108,136 @@ public synchronized void stop() {
 	/**
 	 * This method does nothing.
 	 * <p>
-	 * Subclasses should override this method in order to clear any internal state.
+	 * Subclasses should override this method in order to clear any internal
+	 * state.
 	 */
 	@Override
 	public void clear() {
 	}
+
+	protected void startObserve(final Request request) {
```

Comment:
If this method is not intended to be overridden by subclasses, we should declare it `final`